homeassistant/components/media_player/squeezebox.py

```diff
@@ -173,7 +173,7 @@ def update(self):
     def volume_level(self):
         """ Volume level of the media player (0..1). """
         if 'mixer volume' in self._status:
-            return int(self._status['mixer volume']) / 100.0
+            return int(float(self._status['mixer volume'])) / 100.0
```

Comment:
Okay. I was trying to be consistent with:

```
    def media_duration(self):
        """ Duration of current playing media in seconds. """
        if 'duration' in self._status:
            return int(float(self._status['duration']))
```

Should they both be changed to just float? I can do a pull request for both if needed.